developer-docs-site/docs/nodes/full-node/run-a-fullnode-on-gcp.md

```diff
@@ -138,7 +138,8 @@ Once Terraform apply finished, you can follow this section to validate your depl
 
 4. Check REST API, make sure the ledge version is increasing.
   ```
-  $ curl http://<IP>
+  $ curl http://<IP>/v1
+  # Example: curl http://104.198.36.142/v1
```

Comment:
nit: probably not too effective to include this example, as it's pretty clear that it's the same IP as collected from the previous step. and the IP address in this example might not always work

Reply:
Good point. I will tweak it to say "Example syntax".